iridium-core-server/src/main/java/software/iridium/api/service/AuthorizationService.java

```diff
@@ -380,10 +380,6 @@ public AccessTokenResponse exchange(
             accessTokenRepository.save(
                 accessTokenInstantiator.instantiate(authorizationCode.getIdentityId()));
 
-        final var refreshToken = refreshTokenInstantiator.instantiate(accessToken.getAccessToken());
-        accessToken.setRefreshToken(refreshToken);
-        refreshToken.setAccessToken(accessToken);
```

Comment:
We need to keep the association from the `refreshToken` to the `accessToken`.  

e.g. 
` refreshToken.setAccessToken(accessToken);`

Reply:
The same code is written in AccessTokenInstantiator instantiate function